templates/terraform/command-center-project/service-accounts.tf

```diff
@@ -0,0 +1,11 @@
+module command_center_service_account {
+  source = "/templates/google-sa"
+  providers = {
+    google.target = google.target
+  }
+
+  account_id = var.service_account_id
```

Comment:
Hard-coded or not, I'd have the value here also reflect that this is a "GKE runner" account instead of a generic "command center" account.

Reply:
It looks like terraform doesn't want this id to be longer than 28 characters. @danxmoran does `command-center-gke-runner_sa` (exactly 28 characters) sound okay to you?

Reply:
I'd even leave off the `_sa` suffix. The rest of the email address that's generated for the account makes it clear it's a service account.